### PR TITLE
Make the EnvironmentVariables tests work on UAP.

### PR DESCRIPTION
**Pull request by AtsushiKan**

### CODE REVIEW

File: src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs [hunk 1]
```diff
@@ -85,9 +83,6 @@ public void Default(EnvironmentVariableTarget target)
 
 
         [Theory]
```
Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs]: You removed all of the inputs from this theory.  The test will no longer run anything.

File: src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs [hunk 2]
```diff
@@ -112,9 +107,6 @@ public void ModifyEnvironmentVariable(EnvironmentVariableTarget target)
         }
 
         [Theory]
```
Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs]: You removed all of the inputs from this theory.  The test will no longer run anything.

File: src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs (hunk 1 above)
Comment (AtsushiKan) [inline src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs]: Fixed.

File: src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs (hunk 2 above)
Comment (AtsushiKan) [inline src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs]: Fixed.

**Pull request merged**